schema/project-level/project-package-schema.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "http://standard.open-contracting.org/infrastructure/beta/project-package-schema.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Schema for an OC4IDS project package",
+  "description": "A package of OC4IDS projects. Note that all projects within a package must have a unique project identifier.",
+  "type": "object",
+  "required": [
+    "uri",
+    "publisher",
+    "publishedDate",
+    "projects",
+    "version"
+  ],
+  "properties": {
+    "uri": {
+      "title": "Package identifier",
+      "description": "The URI of this package that identifies it uniquely in the world. Recommended practice is to use a dereferenceable URI, where a persistent copy of this package is available.",
+      "type": "string",
+      "format": "uri"
+    },
+    "publishedDate": {
+      "title": "Published date",
+      "description": "The date that this package was published. If this package is generated 'on demand', this date should reflect the date of the last change to the underlying contents of the package.",
+      "type": "string",
+      "format": "date-time"
+    },
+    "version": {
+      "title": "OC4IDS schema version",
+      "description": "The version of the OC4IDS schema used in this package, expressed as major.minor[-tag]. For example: 1.0-beta or 1.1",
+      "type": "string"
+    },
+    "projects": {
+      "title": "Projects",
+      "description": "A list of projects included in this package, in OC4IDS format",
```

Comment:
Add period.

Reply:
Updated in https://github.com/open-contracting/infrastructure/pull/89/commits/9c694db3cc9ae0626ea4a1330443c8b2f7f2383f